data/mods/My_Sweet_Cataclysm/sweet_monsters.json

```diff
@@ -76,7 +76,7 @@
     "id": "mon_marshmallow_buff",
     "type": "MONSTER",
     "name": { "str": "marshmallow buff" },
-    "description": "A muscular body made of marshmallow, proudly striding towards an unknown goal.  Yummy!",
+    "description": "A muscular body made of marshmallow, proudly striding towards an unknown goal.  It couldn't hurt to take a bite, right?",
```

Comment:
Unless I mispelled yummy or something I'd rather keep the first version. The idea being that 1) it's made of marshmallow and 2) It's a very fit looking marshmallow if you know what I mean.